packages/sections/src/variant/EVA/Body.tsx

```diff
@@ -17,2948 +18,203 @@ import { definition } from ".";
+                        All reported phenotypes:
+                      </Typography>
+                      <Typography variant="caption" display="block">
+                        {cohortPhenotypes.map(cp => (
+                          <div key={cp}>{cp}</div>
+                        ))}
+                      </Typography>
+                    </>
+                }
               </>
             }
             showHelpIcon
           >
-            {/* !! ANY !! as any needed below since onClick sig expects no args !! */}
-            <Link to={`/disease/${disease_id}`} onClick={onLinkClick as any}>
-              {disease_name}
+            <Link to={`/disease/${disease.id}`}>
+              {disease.name}
             </Link>
-        </Tooltip>
-        )
-      },
-      exportLabel: "Disease/Phenotype",
-      exportValue: disease_name => disease_name,
-      filterValue: disease_name => disease_name,
-    },
-    {
-      id: "studyId",
-      label: "ClinVar ID",
-      renderCell: ({ studyId }) =>
-        studyId ? (
-          <Link external to={`https://www.ncbi.nlm.nih.gov/clinvar/${studyId}`}>
-            {studyId}
+          </Tooltip>
+        : <Link to={`/disease/${disease.id}`}>
+            {disease.name}
           </Link>
-        ) : (
-          naLabel
-        ),
-      exportLabel: "ClinVar ID",
     },
-    {
-      id: "clinicalSignificances",
-      label: "Clinical significance",
-      renderCell: ({ clinicalSignificances }) => {
-        if (!clinicalSignificances) return naLabel;
-        if (clinicalSignificances.length === 1) return sentenceCase(clinicalSignificances[0]);
-        if (clinicalSignificances.length > 1)
-          return (
-            <ul
-              style={{
-                margin: 0,
-                padding: 0,
-                listStyle: "none",
-              }}
-            >
-              {clinicalSignificances.map(clinicalSignificance => (
-                <li key={clinicalSignificance}>{sentenceCase(clinicalSignificance)}</li>
-              ))}
-            </ul>
-          );
+    exportValue: ({ disease }) => disease.name,
+    filterValue: ({ disease }) => disease.name,
+  },
+  {
+    id: "studyId",
+    label: "ClinVar ID",
+    renderCell: ({ studyId }) =>
+      studyId ? (
+        <Link external to={`https://www.ncbi.nlm.nih.gov/clinvar/${studyId}`}>
+          {studyId}
+        </Link>
+      ) : (
+        naLabel
+      )
+  },
+  {
+    id: "clinicalSignificances",
+    label: "Clinical significance",
+    renderCell: ({ clinicalSignificances }) => {
+      if (!clinicalSignificances || clinicalSignificances.length === 0) {
         return naLabel;
-      },
-      filterValue: ({ clinicalSignificances }) => clinicalSignificances.join(),
+      }
+      if (clinicalSignificances.length === 1) {
+        return sentenceCase(clinicalSignificances[0]);
+      }
+      return (
+        <ul
```

Comment:
why do we need `ul` and `li` tags here ? i can find example in preview link.

Reply:
we can replace `ul` and `li` with `Box` as it will take care of showing element in next line.

Reply:
Done